src/app/contexts/ServiceContext/index.test.jsx

```diff
@@ -1,5 +1,5 @@
 import React, { useContext } from 'react';
-import { cleanup, render, waitFor } from '@testing-library/react';
+import { cleanup, render, waitFor, act } from '@testing-library/react';
```

Comment:
We can refactor the `waitFor` tests in this file to use act.

Reply:
Could you please link the issue?

Reply:
Just created 🙈 

https://github.com/bbc/simorgh/issues/7019